flink-rpc/flink-rpc-akka/src/test/java/org/apache/flink/runtime/rpc/akka/ContextClassLoadingSettingTest.java

```diff
@@ -107,49 +108,87 @@ void shutdown() throws InterruptedException, ExecutionException, TimeoutExceptio
     void testAkkaRpcService_ExecuteRunnableSetsFlinkContextClassLoader()
             throws ExecutionException, InterruptedException {
         final CompletableFuture<ClassLoader> contextClassLoader = new CompletableFuture<>();
-        akkaRpcService.execute(
-                () -> contextClassLoader.complete(Thread.currentThread().getContextClassLoader()));
+        akkaRpcService
+                .getScheduledExecutor()
+                .execute(
+                        () ->
+                                contextClassLoader.complete(
+                                        Thread.currentThread().getContextClassLoader()));
         assertIsFlinkClassLoader(contextClassLoader.get());
     }
 
     @Test
-    void testAkkaRpcService_ExecuteCallableSetsFlinkContextClassLoader()
+    void testAkkaRpcService_ScheduleCallableSetsFlinkContextClassLoader()
```

Comment:
We should actually remove the callable variant from the scheduled executor because its not used and can be easily replicated like in the test case above.